package.json

```diff
@@ -38,5 +39,8 @@
   "collective": {
     "type": "opencollective",
     "url": "https://opencollective.com/systemjs"
+  },
+  "dependencies": {
+    "node-fetch": "^2.6.0"
```

Comment:
Can we make this a devDependency and use `ncc` to ensure `system-node` is a single file?

Reply:
done with rollup